src/Umbraco.Infrastructure/Migrations/Upgrade/V_16_2_0/AddLastSyncedTable.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_16_2_0;
+
+public class AddLastSyncedTable : AsyncMigrationBase
+{
+    public AddLastSyncedTable(IMigrationContext context) : base(context)
```

Comment:
```suggestion
    public AddLastSyncedTable(IMigrationContext context)
        : base(context)
```

base constructor should be place on its own line :smile: